trademarks.md

```diff
@@ -89,6 +89,14 @@ In those cases, you may use the word "Jupyter" or the unaltered logos to indicat
 without our prior approval.
 This is true both for non-commercial and commercial uses.
 
+Use of stills, videos and screenshots of the Jupyter applications (included but
+not limited to: notebook, jupyterLab, jupyterHub, nbconvert, IPython) as well as
+Jupyter Websites are allowed. Any stills, videos, and screenshots of a specific
+notebook document must be permitted by the license governing that particular
+document and are not subject to this approval, Any picture containing a person
+is subject to this person approval. This approval is worldwide, irrevocable, in
+perpetuity, on digital and mobile and for industrial/trade uses.
+
```

Comment:
I would suggest slightly different language with the same intent.

```
Use of stills, videos, and screenshots of the Jupyter applications (including but
not limited to: Notebook, JupyterLab, JupyterHub, nbconvert, IPython) as well as
Jupyter websites are allowed. Any stills, videos, and screenshots of a specific
notebook document must be permitted by the license governing that particular
document and are not subject to this approval. This approval is worldwide,
irrevocable, and in perpetuity both for non-commercial and commercial uses.
Any picture containing a person is subject to that individual's approval.
```

Reply:
I think we don't need to cover the disclaimers here - we're only giving permission about the Jupyter trademark, so asserting rights about other documents or a person's approval is going beyond what we have scope to do. We could just have a blanket statement that this approval is for the Jupyter trademarks, and does not cover other possible restrictions such as document licenses, etc.

I'd love to have the NumFOCUS (or other lawyer) look at the language about the approval to make sure we used the right technical/legal terms.

Reply:
> I'd love to have the NumFOCUS (or other) lawyer look at the language about the approval to make sure we used the right technical/legal terms.

+1 on this one, I feel this needs some lawyer's eyes to make sure we are doing it properly (on lawyer terms...)

Reply:
I agree, we are getting into the territory of writing a document that has legal consequences. Can someone reach out to Leah Silen from NumFOCUS and ask the NumFOCUS lawyer to have a look? Also, I would like to know if this additional language is even needed. Why aren't these things covered by the relevant licenses we already have?

Reply:
I think that's because some of those documents when screenshot show the Jupyter Logo, and why we want to clarify that it's ok to show it in this context.